src/Engine/ProtoCore/Reflection/MirrorData.cs

```diff
@@ -169,6 +169,9 @@ public IEnumerable<MirrorData> GetElements()
                     return null;
 
                 var array = runtimeCore.Heap.ToHeapObject<DSArray>(svData);
+                if (array == null)
+                    return null;
```

Comment:
I wouldn't recommend changing this method as if this case occurred before, it would have been addressed by now.

Reply:
Ah ... @aparajit-pratap  @mjkkirschner  I will then revert back to my original solution using try/catches within my own code. I can guarantee you that this is an issue, though, and I can demonstrate it for you, if you need me to.

Reply:
In that case, I would be interested in seeing that demo. If what you're saying is true, I guess there's no harm in adding the `null` check. It's just surprising to me that we didn't catch this before. I'll try to sync up later.

Reply:
Here is a bit of an awkward gif to showcase the issue. I have temporarily added a try/catch inside the ``GetElements()`` of MirrorData. It was done using a graph called DynamoOnboardingGuide_HouseCreationDS.dyn which Sol Amour provided me with and I can send out.

The error is thrown when trying to operate on ``array.Values`` with ``array`` being null. 

![mirrordata_nullexception](https://user-images.githubusercontent.com/5354594/173921393-f6540589-0b2c-4c2f-9aa0-b616c5861c2c.gif)